nservicebus/router/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Router
+summary: How to connect parts of the system that use different transports 
+component: Router
+related:
+- samples/azure/azure-service-bus-msmq-bridge
+- samples/msmq/sql-bridge
+reviewed: 2018-05-04
+---
+
+`NServiceBus.Router` is a universal component that connects parts of an NServiceBus-based solution that otherwise cannot talk to each other (e.g. because they use different transport or transport settings or because of physical barriers). 
```

Comment:
> that otherwise cannot talk to each other (e.g. because they use different transport or transport settings

I think it should be

> that otherwise could not talk to each other (e.g. because they use different transports or transport settings